.circleci/config.yml

```diff
@@ -56,6 +56,14 @@ aliases:
       - /opt/ndk
     key: v1-android-ndk-r10e-{{ .Branch }}
 
+  - &restore-cache-watchman
+    keys:
+      - v1-watchman-v4.9.0-{{ .Branch }}
```

Comment:
It'd be useful to have some variables that define which version of watchman, Buck, etc we'd like to use.